### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by aThorp96**
Changes
- Renamed some functions to be clearer
- Changed style to use pydoc comments
- Removed magic numbers
- Added draw functions for point and points

### CODE REVIEW

File: uart.py [hunk 1]
```diff
@@ -4,65 +4,162 @@
...
+    canvas = None
+
+    # points of the four corners of the canvas (in robot arm coords)
+    canvas_corners = None
+
+    # contains the warped image of
+    ptransform = None
+
+    # transformation matrix
+    M = None
```
Comment (JacobJustice) [inline uart.py]: scrap this alongside the methods that use it.

File: uart.py [hunk 2]
```diff
@@ -154,24 +261,39 @@ def saveCoordsToFile(self):
...
+            num_float = float(inp)
+            return True
+        except:
+            return False
+
+    """
+    GET M
+    """
+
+    def get_m(self, width, height):
```
Comment (JacobJustice) [inline uart.py]: scrap this alongside the methods that use it

File: uart.py [hunk 3]
```diff
@@ -180,64 +302,95 @@ def get_m(self):
...
-#    xytoxyz
-#
-    def xy_to_xyz(self,xy):
-        xyz = [xy[0],xy[1],1]
+    """
+    xytoxyz
+    !!DEPRICATED: use xy_to_xyz_geom!!
+    """
+
+    def xy_to_xyz(self, xy):
```
Comment (JacobJustice) [inline uart.py]: scrap this alongside the methods that use it

**Review by JacobJustice — COMMENTED** [review]
test.py isn't being used and can be deleted

File: uart.py (hunk 1 above)
Comment (aThorp96) [reply uart.py]: The M function?

**Review by JacobJustice — COMMENTED** [review]
Didn't delete test.py but otherwise good

**Review by JacobJustice — APPROVED** [review]
Woooooo

**Pull request merged**